Add Bluefin7K Aggregator dimesion-adaptor and deprecate 7K Aggregator

tomkapa opened the pull request:
# PR Description
7K Aggregator has partnered with Bluefin to launch a new aggregator: Bluefin7K Aggregator.

## Related Links:

Previous PR: https://github.com/DefiLlama/dimension-adapters/pull/3410

Commit reference: https://github.com/DefiLlama/defillama-server/commit/1f8c10a0984616eb55ae4884141da03542395295

Announcement: https://x.com/bluefinapp/status/1929596937181032908

# Background
Previously, the original 7K Aggregator was rebranded on DefiLlama to Bluefin7K. However, the 7K team has decided to:
- Preserve the historical volume data of the original 7K Aggregator, and
- Launch Bluefin7K Aggregator as a new, independent aggregator with its own history starting from the official launch date.

# Changes in This PR
1. Adds a new Bluefin7K Aggregator as an independent entry.
2. Updates the existing 7K Aggregator to stop recording further volume data.

The `start` for 7K Aggregator is explicitly set to 2025-06-03 to avoid history entries showing volume = 0, ensuring data consistency.

llamabutler commented:
The 7k-aggregator adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators 7k-aggregator
    
    🦙 Running 7K-AGGREGATOR adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 15 Jun 2025 06:43:23 GMT
    End Date:	Mon, 16 Jun 2025 06:43:23 GMT
    ---------------------------------------------------
    
    SUI 👇
    Backfill start time: 3/6/2025
    Daily volume: 0
    End timestamp: 1750056202 (2025-06-16T06:43:22.000Z)

llamabutler commented:
The bluefin7k-aggregator adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators bluefin7k-aggregator
    
    🦙 Running BLUEFIN7K-AGGREGATOR adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 15 Jun 2025 06:43:25 GMT
    End Date:	Mon, 16 Jun 2025 06:43:25 GMT
    ---------------------------------------------------
    
    SUI 👇
    Backfill start time: 3/6/2025
    Daily volume: 115.21 M
    End timestamp: 1750056204 (2025-06-16T06:43:24.000Z)

llamabutler commented:
The 7k-aggregator adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators 7k-aggregator
    
    🦙 Running 7K-AGGREGATOR adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 17 Jun 2025 00:34:14 GMT
    End Date:	Wed, 18 Jun 2025 00:34:14 GMT
    ---------------------------------------------------
    
    SUI 👇
    Backfill start time: 28/6/2024
    Daily volume: 60.80 M
    End timestamp: 1750206853 (2025-06-18T00:34:13.000Z)

tomkapa commented:
Hi @noateden ,

Just to clarify the reasoning: the previous renaming of 7K Aggregator into Bluefin7K Aggregator was discussed and aligned between both teams — 7K and Bluefin.

From the 7K side, although we no longer operate the 7K Aggregator, we believe it's important to preserve our historical branding and trading volume data under the original name. Therefore, rather than rebranding the existing aggregator, both teams agreed that 7K and Bluefin7K should be treated as separate, independent entities on DefiLlama.

As a result, the Bluefin7K Aggregator — owned and managed by the Bluefin team — has been launched as a new aggregator, with its volume tracked starting from the official announcement date. Meanwhile, the 7K Aggregator, owned by the 7K team, has been archived (disabled) to retain its historical data without continuing to record new volume.

This change is reflected in [PR #9993](https://github.com/DefiLlama/defillama-server/pull/9993), where I reverted the name of the original aggregator (with disabled = true) and introduced a new entry for Bluefin7K Aggregator.

Please note: the name Bluefin7K is set as part of the partnership agreement, and Bluefin is required to use this name for their aggregator for the time being. It may be updated in the future once the contractual period ends.

llamabutler commented:
The 7k-aggregator adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators 7k-aggregator
    
    🦙 Running 7K-AGGREGATOR adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 17 Jun 2025 05:58:35 GMT
    End Date:	Wed, 18 Jun 2025 05:58:35 GMT
    ---------------------------------------------------
    
    SUI 👇
    Backfill start time: 28/6/2024
    Daily volume: 0
    End timestamp: 1750226314 (2025-06-18T05:58:34.000Z)

tomkapa commented:
@noateden could you help me review https://github.com/DefiLlama/defillama-server/pull/9993 also

tomkapa commented:
hi @noateden , do you know why we can’t search for 7K Aggregator on DefiLlama?
